### ISSUE TITLE
[🐛 Bug] SyntaxError : Walrus Operator used with python3.7

### ISSUE THREAD

**Issue by tomitomas** [SeleniumHQ/selenium#13310]
### What happened?

Hi, 

I tried to use selenium on a Debian 10 system, with Python 3.7.3

install the package `pip3 install 'selenium'` => OK
version 4.11.2 of selenium is installed

then I run a python script to open a Chrome page, and an exception at the very first step is catched when the lib is imported
showing usage of Walrus Operator (:=) while the script is launched with python 3.7

the Walrus Operator is available since python 3.8, which make the process failed

### How can we reproduce the issue?

```shell
installation script : 
https://github.com/tomitomas/plugin-noip/blob/dev/resources/install_apt.sh

python script used :
https://raw.githubusercontent.com/tomitomas/plugin-noip/dev/resources/noip-renew.py
```

### Relevant log output

```shell
File "/var/www/html/plugins/noip/core/class/../../resources/noip-renew.py", line 16, in <module>
    from selenium import webdriver
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/__init__.py", line 20, in <module>
    from .chrome.webdriver import WebDriver as Chrome  # noqa
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/chrome/webdriver.py", line 18, in <module>
    from selenium.webdriver.chromium.webdriver import ChromiumDriver
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/chromium/webdriver.py", line 19, in <module>
    from selenium.webdriver.common.driver_finder import DriverFinder
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/common/driver_finder.py", line 22, in <module>
    from selenium.webdriver.common.selenium_manager import SeleniumManager
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/common/selenium_manager.py", line 44
    if (path := os.getenv("SE_MANAGER_PATH")) is not None:
             ^
SyntaxError: invalid syntax
```

### Operating System

Debian 10

### Selenium version

4.11.2

### What are the browser(s) and version(s) where you see this issue?

Chrome 89

### What are the browser driver(s) and version(s) where you see this issue?

ChromeDrive 89

**Comment by titusfortner** [comment]
Minimum version of Python for Selenium is 3.8
https://github.com/SeleniumHQ/selenium/blob/trunk/py/docs/source/index.rst?plain=1#L24-L27

**Comment by DManowitz** [comment]
@titusfortner, [setup.py](https://github.com/SeleniumHQ/selenium/blob/trunk/py/setup.py) indicates `'python_requires': '~=3.7'`.  I ran into this bug with selenium 4.15.1.  In particular, the offending code is here:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\...\lib\site-packages\selenium\webdriver\__init__.py", line 20, in <module>
    from .chrome.webdriver import WebDriver as Chrome  # noqa
  File "C:\...\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 18, in <module>
    from selenium.webdriver.chromium.webdriver import ChromiumDriver
  File "C:\...\lib\site-packages\selenium\webdriver\chromium\webdriver.py", line 19, in <module>
    from selenium.webdriver.common.driver_finder import DriverFinder
  File "C:\...\lib\site-packages\selenium\webdriver\common\driver_finder.py", line 22, in <module>
    from selenium.webdriver.common.selenium_manager import SeleniumManager
  File "C:\...\lib\site-packages\selenium\webdriver\common\selenium_manager.py", line 44
    if (path := os.getenv("SE_MANAGER_PATH")) is not None:
             ^
SyntaxError: invalid syntax
```

**Comment by titusfortner** [comment]
Fixed — b556ac3d6d94db56a0e3385a2c0de29c030ebb7e

@isaulv / @symonk / @AutomatedTester I'm not sure what all is supposed to go in the classifiers section. Do we need to put each supported python version in there?
